### PR TITLE
Add loft intergration

### PR DESCRIPTION
**Pull request by guymenahem**

### CODE REVIEW

File: docs/Integrations/Loft.md [hunk 1]
```diff
@@ -0,0 +1,266 @@
+# Loft Integration
+
+The Loft integration enables you to install a Komodor agent automatically on every vcluster created by Loft.
+The vcluster appears in Komodor's UI like a regular cluster and allows users to access, detect, invetigate and remediate their own vclusters independently.
+You can see the full potential of the Loft <> Komodor integration in this [Live Demo](https://www.linkedin.com/video/event/urn:li:ugcPost:7004685584732471296/).
+ג
```
Comment (nirsht) [inline docs/Integrations/Loft.md:6]: Hebrew

File: docs/Integrations/Loft.md [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+In the Demo, there is a detailed exaplanation on how to configure it using the Loft UI.
```
Comment (nirsht) [inline docs/Integrations/Loft.md:7]:
```suggestion
In the Demo, there is a detailed explanation on how to configure it using the Loft UI.
```

File: docs/Integrations/Loft.md [hunk 3]
```diff
@@ -0,0 +1,266 @@
+# Loft Integration
+
+The Loft integration enables you to install a Komodor agent automatically on every vcluster created by Loft.
+The vcluster appears in Komodor's UI like a regular cluster and allows users to access, detect, invetigate and remediate their own vclusters independently.
```
Comment (nirsht) [inline docs/Integrations/Loft.md:4]:
```suggestion
The vcluster appears in Komodor's UI like a regular cluster and allows users to access, detect, investigate and remediate their own vclusters independently.
```

File: docs/Integrations/Loft.md (hunk 1 above)
Comment (guymenahem) [reply docs/Integrations/Loft.md:6]: What do you mean?

Comment (nirsht) [reply docs/Integrations/Loft.md:6]: there is `ג`

**Review by nirsht — APPROVED** [review]

**Pull request merged**